syntaxes/sassql.tmLanguage.json

```diff
@@ -0,0 +1,674 @@
+          "name": "string.quoted.single.sql",
+          "patterns": [
+            {
+              "include": "#string_escape"
+            }
+          ]
+        },
+        {
+          "captures": {
+            "1": {
+              "name": "punctuation.definition.string.begin.sql"
+            },
+            "2": {
+              "name": "punctuation.definition.string.end.sql"
+            }
+          },
+          "comment": "this is faster than the next begin/end rule since sub-pattern will match till end-of-line and SQL files tend to have very long lines.",
+          "match": "(`)[^`\\\\]*(`)",
+          "name": "string.quoted.other.backtick.sql"
+        },
+        {
+          "begin": "`",
+          "beginCaptures": {
+            "0": {
+              "name": "punctuation.definition.string.begin.sql"
+            }
+          },
+          "end": "`",
+          "endCaptures": {
+            "0": {
+              "name": "punctuation.definition.string.end.sql"
+            }
+          },
+          "name": "string.quoted.other.backtick.sql",
+          "patterns": [
+            {
+              "include": "#string_escape"
+            }
+          ]
+        },
+        {
+          "captures": {
+            "1": {
+              "name": "punctuation.definition.string.begin.sql"
+            },
+            "2": {
+              "name": "punctuation.definition.string.end.sql"
+            }
+          },
+          "comment": "this is faster than the next begin/end rule since sub-pattern will match till end-of-line and SQL files tend to have very long lines.",
+          "match": "(\")[^\"#]*(\")",
+          "name": "string.quoted.double.sql"
+        },
+        {
+          "begin": "\"",
+          "beginCaptures": {
+            "0": {
+              "name": "punctuation.definition.string.begin.sql"
+            }
+          },
+          "end": "\"",
+          "endCaptures": {
+            "0": {
+              "name": "punctuation.definition.string.end.sql"
+            }
+          },
+          "name": "string.quoted.double.sql",
+          "patterns": [
+            {
+              "include": "#string_interpolation"
+            }
+          ]
+        },
+        {
+          "begin": "%\\{",
+          "beginCaptures": {
+            "0": {
+              "name": "punctuation.definition.string.begin.sql"
+            }
+          },
+          "end": "\\}",
+          "endCaptures": {
+            "0": {
+              "name": "punctuation.definition.string.end.sql"
+            }
+          },
+          "name": "string.other.quoted.brackets.sql",
+          "patterns": [
+            {
+              "include": "#string_interpolation"
+            }
+          ]
+        }
+      ]
+    },
+    "saskeywords": {
+      "patterns": [
+        {
+          "match": "(run|quit|%mend)(\\s|/\\*.*\\*/|\\*[^;]*;)*;$",
+          "name": "keyword.section.sas"
+        }
+      ]
+    },
+    "sas-macro-put": {
```

Comment:
nit: how about more consistent names? for example
sas-keywords
sas-macro-put
sas-macro-quotes